### PR TITLE
Add BBVA Compass

### PR DESCRIPTION
**Pull request by jmeridth**
- no 2fa
- image included

### CODE REVIEW

**Comment by stephengroat** [comment]
@jmeridth could you [look at the travis logs](https://travis-ci.org/2factorauth/twofactorauth/jobs/355403074#L1010-L1023)? there's a few errors that need to be dealt with before this is g2g, let us know if you need any help!

**Comment by jmeridth** [comment]
@stephengroat definitely.  Looking.

**Comment by jmeridth** [comment]
@stephengroat :green_heart:

**Comment by stephengroat** [comment]
@jmeridth thanks for the work!

as a rule of thumb, we try to avoid having tons of text in the image icons (at 32x32 px, it's basically impossible to read).

i found the bbva compass favicon and scaled it to 32x32 which i think would be better than what's currently in there, do you know of any other logos they have with no text at all?

bbvacompass

**Comment by jmeridth** [comment]
@stephengroat awesome. thank you.

**Comment by jmeridth** [comment]
@stephengroat done.

**Review by stephengroat — DISMISSED** [review]

File: _data/banking.yml [hunk 1]
```diff
@@ -127,6 +127,13 @@ websites:
       img: bbt.png
       tfa: No
 
+    - name: BBVA Compass
+      url: https://www.bbvacompass.com/
+      img: bbvacompass.png
+      tfa: No
+      facebook: bbvacompass
+      twitter: bbvacompass
```
Comment (Carlgo11) [inline _data/banking.yml]: Please capitalize the twitter tag so that it matches their twitter handle. `BBVACompass`

Thank you!

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by jmeridth** [comment]
@Carlgo11 done.

**Review by Carlgo11 — APPROVED** [review]

**Comment by Carlgo11** [comment]
@jmeridth Great! Thanks :smile:

**Review by mxxcon — APPROVED** [review]

**Pull request merged**